daemon/peer.c

```diff
@@ -341,13 +340,16 @@ static void peer_breakdown(struct peer *peer)
 	
 	/* If we have a closing tx, use it. */
 	if (peer->closing.their_sig) {
+		const struct bitcoin_tx *close = mk_bitcoin_close(peer, peer);
 		log_unusual(peer->log, "Peer breakdown: sending close tx");
-		broadcast_tx(peer, bitcoin_close(peer), NULL);
+		broadcast_tx(peer, close, NULL);
+		tal_free(close);
```

Comment:
Checking `broadcast_tx` it actually `tal_steal`s `close` and assigns it to a `struct outgoing_tx`, which after `tal_free` is a dangling pointer.